src/infrastructure/note.repository.ts

```diff
@@ -87,6 +87,8 @@ export default class NoteRepository implements NoteRepositoryInterface {
     const note: Note = {
       id: response.id,
       content,
+      createdAt: new Date().toISOString(),
+      updatedAt: new Date().toISOString(),
```

Comment:
it seems unexpected. You don't need to pass there properties to create a Note. Backend is responsible for that.

Reply:
its important

Reply:
this PR should not be merged with this, because it overrides the business logic